npk/src/manifest.rs

```diff
@@ -167,6 +146,9 @@ pub enum Mount {
     Tmpfs { size: u64 },
 }
 
+pub type CGroup = HashMap<String, String>;
+pub type CGroups = HashMap<String, CGroup>;
+
```

Comment:
A 3. `CGroups` type...we should not have 3 different types for the cgroups-concept